google/cloud/ndb/_cache.py

```diff
@@ -22,6 +24,8 @@
 _LOCK_TIME = 32
 _PREFIX = b"NDB30"
 
+warnings.filterwarnings("always", module=__name__)
```

Comment:
So I have better context here, what is this intended to filter out and why?

Reply:
It's actually unfiltering. The default, for warnings, is to log them once and don't log any duplicates. This is overriding that default so the warnings are shown every time.

Reply:
Interesting! Thank you